site2/docs/security-kerberos.md

```diff
@@ -107,6 +107,7 @@ It is also a choice to have 2 separate JAAS configuration files: the file for br
  - Set `authenticationProviders` to choose `AuthenticationProviderSasl`;
  - Set `saslJaasClientAllowedIds` regex for principal that is allowed to connect to broker. 
  - Set `saslJaasBrokerSectionName` that corresponding to the section in JAAS configuration file for broker.
+ - Set `superUserRoles` that corresponding to the name registered in kdc.
```

Comment:
This need set when `authorizationEnabled` is set to true.
How about adding it at around line 270, which is related to authorization

Reply:
updated. PTAL